src/main/java/net/bramp/ffmpeg/info/FilterPattern.java

```diff
@@ -0,0 +1,75 @@
+package net.bramp.ffmpeg.info;
+
+import net.bramp.ffmpeg.probe.FFmpegStream;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.util.*;
+
+public class FilterPattern {
+    /** Indicates weather this pattern represents a source or a sink and therefore has no other options */
```

Comment:
Typo s/weather/whether/g and later in this file

Reply:
Thanks for the sed pattern. Just pushed the typo fixes